ovirt-imageio.spec.in

```diff
@@ -145,6 +172,53 @@ if systemctl is-active ovirt-imageio-daemon.service >/dev/null; then
     systemctl stop ovirt-imageio-daemon.service
 fi
 
+%if %{with_python38}
+%define python3_pkgversion 38
+%define __python3 /usr/bin/python3.8
+
+%package -n python38-imageio-common
+Summary:   oVirt imageio common resources
+
+# NOTE: keep in sync with automation/build-artifacts.packages
+BuildRequires: gcc
+BuildRequires: python38-devel
+
+# NOTE: keep in sync with automation/check.packages
+Requires:  python38
+
+%description python38-imageio-common
```

Comment:
Please change to `%description -n python38-imageio-common` without the -n it tries to make `ovirt-imageio-python38-imageio-common`

Reply:
Done